### DISCUSSION TITLE
Welcome to nuclei-templates Discussions!

### DISCUSSION THREAD

**Discussion by ehsandeep** [projectdiscovery/nuclei-templates#670] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dwisiswant0** [comment]
Yess!!! 💃

**Reply by ehsandeep** [reply]
@dwisiswant0 🥇💃

**Comment by JackoPrograms** [comment]
Good afternoon!

Could you tell me how to learn to understand the output of nuclei ?
What do all these words mean, where can I find out a poem about it?

![scan](https://github.com/projectdiscovery/nuclei-templates/assets/114219862/1b1afbe9-6fe4-4ec9-b7dc-524435178726)

**Comment by MetzinAround** [comment]
Hi @JackoPrograms. These results are telling you the templates, and thus the vulnerabilities, found in the scan. You can clik the info link there or you can head to [this repos](https://nux.gg/templates) main page and find those templates in there to learn more.
